test/Validator/IsIntTest.php

```diff
@@ -131,4 +132,57 @@ public function testEqualsMessageTemplates()
             $validator
         );
     }
+
+    public function testGetStrict()
+    {
+        $this->assertSame(
+            IsIntValidator::COMPARE_NOT_STRICT,
+            $this->validator->getStrict()
+        );
+
+        $this->validator->setStrict(IsIntValidator::COMPARE_STRICT);
+        $this->assertSame(
+            IsIntValidator::COMPARE_STRICT,
+            $this->validator->getStrict()
+        );
+    }
+
+    public function testSetStrictThrowsInvalidArgumentException()
+    {
+        $this->setExpectedException(Exception\InvalidArgumentException::class);
+
+        $this->validator->setStrict(-1);
+    }
+
+    public function strictIntDataProvider()
```

Comment:
Docblock missing, at least we should have `@return array` (helps for adding PHP 7 return type declarations later)